Makefile

```diff
@@ -132,8 +129,16 @@ release: all deps docs $(SMF_MANIFESTS)
 	(cd $(RELSTAGEDIR) && $(TAR) -I pigz -cf $(ROOT)/$(RELEASE_TARBALL) root site)
 	@rm -rf $(RELSTAGEDIR)
 
-regen_docs:
-	$(NODE) ./tools/gendocs.js docs/static.md lib/endpoints > docs/index.md
+./node_modules/.bin/doctoc:
+	npm install
+
+# Make a table of contents in Markdown docs that are setup to use it.  This
+# changes those files in-place, so one should do this before commit.
+.PHONY: tocs
+tocs: | ./node_modules/.bin/doctoc
```

Comment:
I think it should be `toc`, not the plural `tocs`, i.e. `make toc`

Reply:
Or drop the separate "toc" target and put the `doctoc ...` command(s) under the `doc` target.